docs/components/base/typography.js

```diff
@@ -0,0 +1,47 @@
+import { css } from 'react-emotion';
+import * as Constants from '~/common/constants';
+
+export const h1 = css`
+  font-family: ${Constants.fonts.bold};
+  color: ${Constants.colors.black90};
+  font-size: 52px;
```

Comment:
```suggestion
  font-size: 42px;
```
52px seemed a bit too large on desktop and mobile -- 42px felt more comfortable

Reply:
I made the text match the website, which adheres to a font scale. We can scale down this font stack for the docs if you both think it's too large. I'd just need to scale down all the headers, and then consider if we want to scale down the body text (which is at 16px right now). 

Here's two examples in both directions:
Example of large headers: https://reactnative.dev/docs/intro-react
Example of small headers: https://stripe.com/docs/api

To me, the larger sizes are inviting and make it easier to scan the headers. With that, if it's too much let's scale 'em back.  (cc @brentvatne @tommybru )

Reply:
I'd be open to scaling down just the headers and seeing how the docs look. I think the 16px body text works really well. But I find the H1 a little too large to easily scan.